content/github/importing-your-projects-to-github/importing-source-code-to-github/adding-an-existing-project-to-github-using-the-command-line.md

```diff
@@ -31,27 +31,20 @@ shortTitle: Add a project locally
 1. Initialize the local directory as a Git repository.
 
     ```shell
-    git init -b main
+    git -C <em>path/to/repo</em> init -b main
     ```
 
-1. To create a repository for your project on {% data variables.product.product_name %}, use the `gh repo create` subcommand. Replace `project-name` with the desired name for your repository. If you want your project to belong to an organization instead of to your user account, specify the organization name and project name with `organization-name/project-name`.
+1. Stage and commit all the files in your project 
 
    ```shell
-   gh repo create <em>project-name</em>
+   git -C <em>path/to/repo</em> add . && git -C <em>path/to/repo</em> commit -m “initial commit”
    ```
 
-1. Follow the interactive prompts. Alternatively, you can specify arguments to skip these prompts. For more information about possible arguments, see [the {% data variables.product.prodname_cli %} manual](https://cli.github.com/manual/gh_repo_create).
-1. Pull changes from the new repository that you created. (If you created a `.gitignore` or `LICENSE` file in the previous step, this will pull those changes to your local directory.)
+1. To create a repository for your project on GitHub, use the `gh repo create` subcommand. When prompted, select “Push an existing local repository to GitHub”. Replace the project name with the desired name for your repository. If you want your project to belong to an organization instead of your user account, specify the organization name and project name with `organization-name/project-name`.
```

Comment:
Should we say this instead?:
```suggestion
1. To create a repository for your project on GitHub, use the `gh repo create` subcommand. When prompted, select **Push an existing local repository to GitHub** and enter the desired name for your repository. If you want your project to belong to an organization instead of your user account, specify the organization name and project name with `organization-name/project-name`.
```